cloudbuild-deploy.yaml

```diff
@@ -0,0 +1,25 @@
+steps:
+  - name: 'gcr.io/cloud-builders/docker'
+    args: ['build',
+           '-t', 'gcr.io/$PROJECT_ID/$REPO_NAME:$SHORT_SHA',
+           '-t', 'gcr.io/$PROJECT_ID/$REPO_NAME:$BRANCH_NAME-latest',
```

Comment:
Is there a point to including the branch name here? The service name doesn't include the branch, so it's always just going to run whatever branch ran this most recently...

Reply:
I remember adding branch name previously because someone could invoke `make docker` on a non-`master` branch and it would still get labeled as `latest`. 

In this case, this `cloudbuild` config doesn't work unless started by a Cloud Build Trigger (that is, you can't `--config cloudbuild-deploy.yaml` via command line). The trigger I've setup only runs on `master`, so you're right in that it may be overkill, but it's more descriptive than simply `latest`. Happy to discuss other options.